.github/workflows/build_docs.yaml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - main
+env:
+  PYARROW_IGNORE_TIMEZONE: 1
+  JAVA_HOME: "/usr/lib/jvm/java-11-openjdk-amd64"
+  ALTERYX_OPEN_SRC_UPDATE_CHECKER: False
```

Comment:
We could remove `ALTERYX_OPEN_SRC_UPDATE_CHECKER ` on [line 40](https://github.com/alteryx/woodwork/blob/7cd3a75fb14360b1658e1488fda58b53b8f09cdc/.github/workflows/build_docs.yaml#L40) since we're adding here.

Reply:
Fixed